Torch.API/Managers/IPluginManager.cs

```diff
@@ -14,12 +14,12 @@ public interface IPluginManager : IManager, IEnumerable<ITorchPlugin>
         /// <summary>
         /// Fired when plugins are loaded.
         /// </summary>
-        event Action<IList<ITorchPlugin>> PluginsLoaded;
+        event Action<ICollection<ITorchPlugin>> PluginsLoaded;
```

Comment:
Suggest `IReadOnlyCollection` here, since event handlers shouldn't be changing the list of plugins.